specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -1364,7 +1456,8 @@
         }
       },
       "required": [
-        "kind"
+        "kind",
+        "properties"
```

Comment:
same as above, there is no `properties` here - if a schema further up the inheritance chain has that property, it must be `required` up there

Reply:
Removed, thanks.